### PR TITLE
Preparing release 1.0.0

### PR DESCRIPTION
**Pull request by OscarSpruit**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## New
+- Display payment method icon in front of favorites.
```
Comment (jreij) [inline RELEASE_NOTES.md:2]: Nit: this can be written the same style as the rest

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 
-This is the initial release of Adyen Test Cards for Android 🥳
-
-See the [README](./README.md) for more information on how to install and use the app.
+## Fixed
+- Spaces are no longer included when autofilling a number. Thanks to @dhartwich1991!
```
Comment (jreij) [inline RELEASE_NOTES.md:5]: I think the tag won't work in the md file, when i try to preview it, it just looks like normal text

Comment (OscarSpruit) [reply RELEASE_NOTES.md:5]: I think it will work once we publish the release

File: RELEASE_NOTES.md (hunk 1 above)
Comment (OscarSpruit) [reply RELEASE_NOTES.md]: Updated

**Review by jreij — APPROVED** [review]

**Pull request merged**